### PR TITLE
Invoice sales analyst

### PR DESCRIPTION
**Pull request by chsweet**
Type of Change:

- Update spec tests

What does it implement:

- Tests for all methods in InvoiceRepository
- Tests for all methods in SalesAnalyst 

Any relevant logs, error outputs, etc.:

- Currently passing all tests

### CODE REVIEW

File: spec/invoice_repository_spec.rb
Comment (marlitas) [inline spec/invoice_repository_spec.rb]: Leftover merge issue.

**Review by marlitas — DISMISSED** [review]
Approved after removing merge leftovers

File: spec/invoice_repository_spec.rb
Comment (chsweet) [reply spec/invoice_repository_spec.rb]: Updated and pushed

**Pull request merged**
